src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/slurm/SlurmJobManager.groovy

```diff
@@ -183,9 +190,15 @@ class SlurmJobManager extends GridEngineBasedJobManager {
             Duration runLimit = safelyParseColonSeparatedDuration(jobResult["TimeLimit"])
             Duration runTime = safelyParseColonSeparatedDuration(jobResult["RunTime"])
             jobInfo.runTime = runTime
-            BufferValue memory = safelyCastToBufferValue(jobResult["mem"])
-            Integer cores = withCaughtAndLoggedException { jobResult["NumCPUs"] as Integer }
+            BufferValue memory = safelyCastToBufferValue(jobResult["MinMemoryNode"])
```

Comment:
```suggestion
            // MinMemoryNode and MinCPUsNode are the request of the minimum available
            // on the cluster node. In contrast, for LSF `ResourceSet.memory` and `cores` are **additionally**
            // the maxima, beyond which the job will be killed because too many resources
            // are used.
            BufferValue memory = safelyCastToBufferValue(jobResult["MinMemoryNode"])
```